.copier-answers.yml

```diff
@@ -1,6 +1,6 @@
 # Changes here will be overwritten by Copier
```

Comment:
Is the comment here still true? Should the value be updated upstream somewhere?

Reply:
I believe the comment comes from running copier. The src_path will not be overwritten